docs/tutorials/excel-read-tutorial.md

```diff
@@ -55,7 +55,7 @@ Over the rest of the tutorial, we will normalize this data using a script. First
     }
     ```
 
-5. Now let's read a value from one of the number columns. Add the following code to the end of the script (before the closing `}`):
+5. Now let's read a value from one of the number columns. Add the following code to the end of the script (before the closing `}`).
```

Comment:
```suggestion
5. Now let's read a value from one of the number columns. Add the following code to the end of the script (before the closing `}`).
```
We use second person "you" in MSFT voice. Can we fix that in this article? (Remove pronouns "let's" and "we" and change to "you.")

Reply:
I did a pass on this and the other tutorials for this issue. Should be cleared up now.